lib_eio/fs.ml

```diff
@@ -48,3 +52,15 @@ and virtual dir_with_close = object
   inherit dir
   method virtual close : unit
 end
+
+let read_at_offset (t : #read_at_offset) ofs buf =
```

Comment:
Could just call this `read` and take an optional argument. `eio_linux.mli` uses `file_offset`, e.g.
```suggestion
let read (t : #ro) ?file_offset buf =
```

Reply:
Here I was in two minds because there already is a `Flow.read` which corresponds to the case of `file_offset = None`. So for the moment I renamed this to `pread`, with a *mandatory* `~file_offset` argument (to avoid duplicating the functionality of `Flow.read` which may be a bit confusing).